extensions/kafka/src/main/java/io/deephaven/kafka/AvroImpl.java

```diff
@@ -453,7 +474,11 @@ private static void pushColumnTypesFromAvroField(
                 break;
             case ENUM:
             case STRING:
-                columnsOut.add(ColumnDefinition.ofString(mappedNameForColumn));
+                if (useUTF8Strings) {
+                    columnsOut.add(ColumnDefinition.of(mappedNameForColumn, Type.find(CharSequence.class)));
```

Comment:
We should probably use the more specific type, `org.apache.avro.util.Utf8`. Also, can store the type as a constant.

Reply:
We'll also want to be very careful that the avro deserializer doesn't pool / keep hold of the `Utf8` objects; the javadoc explicitly states it's mutable:

```java
/**
 * A Utf8 string. Unlike {@link String}, instances are mutable. This is more
 * efficient than {@link String} when reading or writing a sequence of values,
 * as a single instance may be reused.
 */
```